<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingToolbox</class>
 <widget class="QDockWidget" name="ProcessingToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Processing Toolbox</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QgsFilterLineEdit" name="searchBox">
      <property name="toolTip">
       <string>Enter algorithm name to filter list</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="algorithmTree">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="modeComboBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
